django/contrib/sessions/backends/signed_cookies.py

```diff
@@ -17,7 +17,9 @@ def load(self):
                 # This doesn't handle non-default expiry dates, see #19201
                 max_age=settings.SESSION_COOKIE_AGE,
                 salt='django.contrib.sessions.backends.signed_cookies')
-        except (signing.BadSignature, ValueError):
+        except Exception:
+            # BadSignature, ValueError, or unpickling exceptions. If any of
+            # these happen, reset the session.
```

Comment:
My justification is that we are catching `Exception` in similar places of other cache backends. If it ends up hiding actual problems, maybe a solution could be to add debug logging.